src/tests/roc_pipeline/target_stdio/test_port.cpp

```diff
@@ -186,5 +186,42 @@ TEST(port, bad_port_number) {
     CHECK(!parse_port(Port_AudioSource, "rtp:1.2.3.4:999999999999999", port));
 }
 
+TEST(port, multicast_ipv4) {
```

Comment:
We also need a check that if address is multicast, but the `@...` part is missing, the parsing fails. Both for ipv4 and ipv6.

Reply:
We also need to check that both interface name and IP address can be used after `@`.

Reply:
> We also need to check that both interface name and IP address can be used after @.

https://github.com/roc-project/roc/pull/247#discussion_r326856574